app/helpers/admin/claims_helper.rb

```diff
@@ -2,6 +2,10 @@ module Admin
   module ClaimsHelper
     include StudentLoans::PresenterMethods
 
+    def confirming_identity_playbook_url
+      "https://docs.google.com/document/d/1wZh68_RV_FTJLxXIDPr3XFtJHW3vRgiXGaBDUo1Q1ZU"
```

Comment:
Not relevant at all to the code, and not a blocker, but should this be in Confluence, rather than as a Google Doc? Just concious of the risk of documentation being all over the place.

Reply:
That's where the service operations team have put this and other docs. If we think they should be storing those elsewhere then we should tackle that separately and with their OK. I'm not even sure they have Confluence access tbh...

Reply:
Aaaah, if they don't even have Confluence access, then it's probably fine to have this stuff in Google docs.